server/zio-http/src/main/scala/sttp/tapir/server/ziohttp/ZioHttpToResponseBody.scala

```diff
@@ -35,7 +36,11 @@ class ZioHttpToResponseBody extends ToResponseBody[ZStream[Any, Throwable, Byte]
       case RawBodyType.ByteArrayBody       => Stream.fromChunk(Chunk.fromArray(r))
       case RawBodyType.ByteBufferBody      => Stream.fromChunk(Chunk.fromByteBuffer(r))
       case RawBodyType.InputStreamBody     => ZStream.fromInputStream(r).provideLayer(Blocking.live)
-      case RawBodyType.FileBody            => ZStream.fromFile(r.toPath).provideLayer(Blocking.live)
+      case RawBodyType.FileBody            =>
+        val tapirFile = r.asInstanceOf[FileRange]
+        FileChunk.prepare(tapirFile)
+          .map(chunk => ZStream.fromInputStream(chunk).provideLayer(Blocking.live))
+          .getOrElse(ZStream.fromFile(tapirFile.toPath).provideLayer(Blocking.live))
```

Comment:
less probable, but maybe here as well there's an option to get a zio-stream from a file chunk?